src/content/blog/v1.0_release.md

```diff
@@ -0,0 +1,75 @@
+
+<br/><br/>
+The new PyTorch backend allows AutoEmulate to take advantage of PyTorch's powerful features, including (but not limited to):
+
+- Leveraging GPU acceleration, making training and inference much faster.
+- Enabling automatic differentiation via PyTorch's autograd system.
+- Seamlessly integrating AutoEmulate with other tools in the broader ecosystem, enabling end-to-end emulation workflows.
+
+### A new set of Emulator models
+
+AutoEmulate v1.0.0 includes a new set of Emulator models with a variety of different architectures, built on top of PyTorch (and GPyTorch for Gaussian processes). Many of these are custom implementations built by the AutoEmulate team, including ensemble models where we quantify uncertainty by looking at the distribution over outputs, but some are simply wrapped PyTorch functions.
+<br/>
+Additionally, we have retained support for several non-PyTorch models that our userbase has found useful. The table below summarises which emulators available in this release are implemented in PyTorch, support multi-output emulation (MO), use auto-differentiation (AD) for training, or provide predictive uncertainty quantification (UQ).
+
+| Emulator | PyTorch | MO | AD | UQ |
+|----------|-------|---------|----|----|
+| GaussianProcess | ✔️ | ✔️ | ✔️ | ✔️ |
+| GaussianProcessCorrelated | ✔️ | ✔️ | ✔️ | ✔️ |
+| EnsembleMLP | ✔️ | ✔️ | ✔️ | ✔️ |
+| EnsembleMLPDropout | ✔️ | ✔️ | ✔️ | ✔️ |
+| MLP | ✔️ | ✔️ | ✔️ | ❌ |
+| PolynomialRegression | ✔️ | ✔️ | ✔️ | ❌ |
+| RadialBasisFunctions | ✔️ | ✔️ | ❌ | ❌ |
+| RandomForest | scikit-learn  | ✔️ | ❌ | ❌ |
+| SupportVectorMachine | scikit-learn | ❌ | ❌ | ❌ |
+| LightGBM | LightGBM | ❌ | ❌ | ❌ |
+
+### Bayesian model calibration
+
+Bayesian calibration is a method for estimating which input parameters were most likely to produce observed data. An advantage over other calibration methods is that it returns a probability distribution over the input parameters rather than just point estimates.
+Performing Bayesian calibration requires a fit emulator and observations associated with the simulator output. Check out our new [tutorial](<URL>) for more details on how to use this feature.
+
+### Upgraded features
+
+- **Simulator in the loop**: Some features in AutoEmulate such as *active learning* and *history matching* require the user to provide a simulator that can be run by AutoEmulate. AutoEmulate now supports easy integration of simulators into the emulation workflow. This is achieved through a subclassing mechanism of the *Simulator* base class, allowing the users to define a method to run their simulation, taking in the input parameters and returning the output variables. See our [custom simulations tutorial](<URL>) for more details.
```

Comment:
Removing history matching from mention here as it can be used without a simulator as a metric.

```suggestion
- **Simulator in the loop**: Some features in AutoEmulate such as *active learning* require the user to provide a simulator that can be run by AutoEmulate. AutoEmulate now supports easy integration of simulators into the emulation workflow. This is achieved through a subclassing mechanism of the *Simulator* base class, allowing the users to define a method to run their simulation, taking in the input parameters and returning the output variables. See our [custom simulations tutorial](<URL>) for more details.
```